app/controllers/api/v1/authentication_controller.rb

```diff
@@ -190,6 +189,24 @@ def auth_params
       params.require(:user_auth).permit(attributes)
     end
 
+    def render_invalid_mobile
+      attr = I18n.t('activerecord.attributes.user.mobile')
+      reason = @mobile.blank? ? 'blank' : 'invalid'
+      err = I18n.t("activerecord.errors.models.user.attributes.mobile.#{reason}")
+      message = I18n.t('errors.format', attribute: attr, message: err)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.